pkg/cmd/server/server.go

```diff
@@ -159,6 +161,10 @@ func (c *Config) Complete(ctx context.Context) (RunnableServer, error) {
 			dispatchPresharedKey = c.PresharedKey[0]
 		}
 
+		specificConcurrencyLimits := c.DispatchConcurrencyLimits
+		concurrencyLimits := specificConcurrencyLimits.WithOverallDefaultLimit(c.OverallDispatchConcurrencyLimit)
+		log.Debug().EmbedObject(concurrencyLimits).Msg("configured dispatch concurrency limits")
```

Comment:
hrm, I'm wondering if this is better suited for info level. It only happens once on bootstrap, and it's useful information to validate you are getting the expected values in production without bumping to debug level

Reply:
Yeah, I went back and forth on this. @jzelinskie thoughts?

Reply:
I believe the others are printed as info, too. Unless we don't want folks to tinker with this flag, I'd say info is fine.